lib/charms/opensearch/v0/opensearch_backups.py

```diff
@@ -428,11 +442,26 @@ def _can_unit_perform_backup(self, event: ActionEvent) -> bool:
             return False
         return not self.is_backup_in_progress()
 
+    def _format_backup_id(
```

Comment:
why do we need to wrangle 2 different date formats, why not stick to `BACKUP_ID_FORMAT` everywhere?